### PR TITLE
desktop (for mac) remove "button" styling for older release downloads

### PR DESCRIPTION
**Pull request by thaJeztah**
Only show the latest release download links as a CTA button, and older
releases as regular links.

### CODE REVIEW

File: docker-for-mac/release-notes.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ This page contains information about the new features, improvements, known issue
 ## Docker Desktop 3.3.2
 2021-05-03
 
-[Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/63878/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn}
-[Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/63878/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn}
+> [Download for Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/63878/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64)
+> [Download for Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/63878/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64)
```
Comment (thaJeztah) [inline docker-for-mac/release-notes.md:33]: Wasn't sure if we needed to keep the utms on these older links

**Comment by thaJeztah** [comment]
@usha-mandya ptal

File: docker-for-mac/release-notes.md (hunk 1 above)
Comment (usha-mandya) [reply docker-for-mac/release-notes.md:33]: Hmm good question. I don't think we need this for older releases 

@stephen-turner WDYT?

Comment (thaJeztah) [reply docker-for-mac/release-notes.md:33]: It was a bit unclear to me indeed what we're tracking; if we want "any link from docs that led to a download" to be tracked by these, we should update all links on this page. If we're interested in differentiating "CTA" clicks vs "older downloads", perhaps remove them from these.

**Comment by thaJeztah** [comment]
Removed the UTM codes from download links for older releases

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**